src/libraries/Common/src/Interop/Windows/WinSock/Interop.WSADuplicateSocket.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Winsock
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct WSAProtocolChain
```

Comment:
WSAPROTOCOLCHAIN

Reply:
My previous statement is mostly true for structures.